src/utilities/getBMI/getBMI.js

```diff
@@ -0,0 +1,15 @@
+export function getBMI(weight, height) {
+  if (typeof weight !== 'number' || typeof height !== 'number') {
+    return 'please provide weight as a number of kilograms and height as a number in meters';
```

Comment:
If you want to handle the arguments being incorrect, throwing an error is a much better idea than returning a string.